README.md

```diff
@@ -1,36 +1,31 @@
 ![Build](https://github.com/AdoptOpenJDK/openjdk-api-v3/workflows/Build/badge.svg?branch=master) [![codecov](https://codecov.io/gh/AdoptOpenJDK/openjdk-api-v3/branch/master/graph/badge.svg)](https://codecov.io/gh/AdoptOpenJDK/openjdk-api-v3)
 
-# AdoptOpenJDK API
+# Adoptium API
 
-**NOTICE:** AdoptOpenJDK API v1 has now been removed.
-If you are using v1 please move to the latest version (documented below) as soon as possible.
-
-**NOTICE:** [AdoptOpenJDK API v2](https://github.com/AdoptOpenJDK/openjdk-api/blob/master/README.md) has been deprecated and will be removed.
-If you are using v2 please move to the latest version (documented below) as soon as possible.
+**NOTICE:** [AdoptOpenJDK API v3](https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/README.md) has been deprecated as part of the [move to Eclipse Foundation and rebrand to Adoptium](https://blog.adoptopenjdk.net/2021/03/transition-to-eclipse-an-update/).
```

Comment:
```suggestion
**NOTICE:** [AdoptOpenJDK API v3](https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/README.md) has been deprecated as part of the move to Eclipse Foundation and rebrand to Adoptium.
```
We have been requested not to link back to AdoptOpenJDK, so just removing this (otherwise perfectly valid) link.